### PR TITLE
Rails 6.0 Upgrade Task: update content_type to media_type in home_controller and change update_attributes to update in workflows_dump_worker_spec

### PR DESCRIPTION
**Pull request by yuenmichelle1**
https://github.com/zooniverse/panoptes/issues/4097

Describe your change here.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: spec/controllers/home_controller_spec.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
       end
 
       it "should default to html" do
-        expect(response.content_type).to eq("text/html")
+        expect(response.media_type).to eq('text/html')
```
Comment (yuenmichelle1) [inline spec/controllers/home_controller_spec.rb:19]: I question whether we still need these tests, knowing that (per Deprecation warning: 
`Rails 6.1 will return Content-Type header without modification.`). Regardless I updated them.

Comment (camallen) [reply spec/controllers/home_controller_spec.rb:19]: It's probably worth checking how rspec sets up the get request, specifically the `content_type` header. If it's a sound default then we can probably skip this test, perhaps useful https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec#requesting-a-json-response

File: spec/workers/workflows_dump_worker_spec.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@
         tasks: tasks, pairwise: !workflow.pairwise,
         grouped: !workflow.grouped, prioritized: !workflow.prioritized
       }
-      workflow.update_attributes(updates)
+      workflow.update(updates)
```
Comment (camallen) [inline spec/workers/workflows_dump_worker_spec.rb:23]: did this one sneak into this PR? happy for it to stay

**Review by camallen — APPROVED** [review]
LGTM

File: spec/workers/workflows_dump_worker_spec.rb (hunk 2 above)
Comment (yuenmichelle1) [reply spec/workers/workflows_dump_worker_spec.rb:23]: oops very possible that i lumped it in since i was on the spec file anyway.

**Pull request merged**